### ISSUE TITLE
Using `pipe` with release `cljstyle_0.16.626_linux_amd64.zip` replaces non-ASCII-UTF-8 chars with a question mark

### ISSUE THREAD

**Issue by christoph-frick** [greglook/cljstyle#108]
I tried with the usual suspects for "regular" java applications
(`JAVA_OPTIONS=-Dfile.encoding=UTF-8`), but no cigar.

I guess this has to be a build setting for `native-image`; e.g. https://github.com/oracle/graal/issues/2492#issuecomment-654513764

At least EDN is supposed to be encoded as UTF-8
(https://github.com/edn-format/edn?tab=readme-ov-file#general-considerations);
I could not find a proper link to make the argument for CLJ files
though.

```console
% cljstyle version
mvxcvi/cljstyle 0.16.626 (built from 58f88f28d28d8692a2a89e0ff297b5552759c1c5 on 2024-01-06)
% echo $LANG 
en_US.UTF-8
% cat test.edn 
["Copyright © 2024"]
% cat test.edn | cljstyle pipe 
["Copyright ? 2024"]
```

(Note the `?` after the last `Copyright`)

**Comment by greglook** [comment]
This is super annoying; I was able to reproduce this and it should be fixed in the next release!
```
root@b1fdcec24560:/opt/cljstyle# cljstyle version
mvxcvi/cljstyle 0.17.635 (built from bd0a6ff5bbab1b2b0364c99fdcd8d70390e35c5a on 2024-12-05)

root@b1fdcec24560:/opt/cljstyle# cat > test.edn
["Copyright © 2024"]

root@b1fdcec24560:/opt/cljstyle# cljstyle pipe < test.edn 
["Copyright © 2024"]
```

**Comment by greglook** [comment]
Fixed in [0.17.642](https://github.com/greglook/cljstyle/releases/tag/0.17.642)
